components/backup/src/endpoint.rs

```diff
@@ -1469,6 +1506,212 @@ pub mod tests {
         }
     }
 
+    fn generate_test_raw_key(idx: u64, api_ver: ApiVersion) -> String {
+        let mut key = if idx == 100 {
```

Comment:
Nit: a little surprised for this special value. Seems `0` would be better.

Reply:
idx is used as the timestamp, which should be the same as the causal_ts::TestProvider::get_ts  value because we need do checksum. Maybe define a contant in causal_ts::TestProvider and use it here?

Reply:
En... it is more surprised to me...
Seems we should checksum on user key & user value, other than encoded key with timestamp ? Otherwise, how we verify backup checksum for V1 to V2 ?

Reply:
changed to do checksum with user_key/user_value.